### ISSUE TITLE
ios 14.1 support

### ISSUE THREAD

**Issue by hermeng** [master131/iFakeLocation#36]
Hi,i've download the newest release of 1.5 but  I'm still getting "Your device's iOS version is not supported at this time." with iOS 14.1 devices.
btw ，i also tried to create the DeveloperImage folder but still failed. it says cant mount the image.
thank you

**Comment by master131** [comment]
That is quite strange; are you on Windows or macOS? Also, is there anything on your network that would prevent the application from talking to Github (proxy, firewall, etc)?

I am not sure why it would be unable to mount the developer image other if you downloaded it manually. Did you try this image?
https://github.com/mspvirajpatel/Xcode_Developer_Disk_Images/releases/download/14.1/14.1.zip

**Comment by jas0nt373** [comment]
Having the same problem as the OP after I updated iOS to 14.1. No issues or changes before I updated.

**Comment by sween64** [comment]
I was using v1.4.3 without issues until today. I downloaded v1.5.0 and am having the same issue. "Unable to mount developer image." I had previously manually downloaded the developer imager for ios 14.1. When I started v1.5.0 of iFL it downloaded the developer image for me successfully, but the same error occurred. I tried a manual download from the link master131 posted above and again the same error occurred. 

I'm not sure if it's related or not but the iOS 14.1 developer image is not available from [here](https://github.com/haikieu/xcode-developer-disk-image-all-platforms/tree/master/DiskImages/iPhoneOS.platform/DeviceSupport), which is the link in your readme.
